modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/FileParsers.js

```diff
@@ -124,16 +144,23 @@ export function extractFieldsFromJSONL(content) {
 
 export function extractFieldsFromJSON(content) {
 	const jsonArray = content.split('');
-	let parsedJSON;
-
 	jsonArray.shift();
 
+	const jsonfile = JSON.parse(content);
+	const contentLineColumns = jsonfile
```

Comment:
Also Here, please take into account that content may contain partial values which will likely lead the JSON.parse function to fail